chapters/annotations.tex

```diff
@@ -189,14 +170,39 @@ \subsubsection{Axis Properties}\label{axis-properties}
 When an axis bound isn't provided, the tool computes one automatically.
 
 The Modelica tool is responsible for showing the unit used for values at the
-axis tick marks, so the axis \lstinline!label! shall not be used to convey this
-information.
+axis tick marks, so the axis \lstinline!label! shall not contain the unit.
+
+If a tool does not recognize the \lstinline!unit!, it is recommended to issue a
+warning and treat the \lstinline!unit! as if it was not specified.
```

Comment:
I realize that this isn't really an option as long as the `unit` is mandatory.  However, I believe that having a mandatory `Axis.unit`, which in turn causes `Plot.x` and `Plot.y` to be mandatory, is a problem that needs to be resolved separately, and that we open up a separate PR for how this should be done.

Reply:
That separate PR is now opened as #2632.